ENV/lib/python2.7/site-packages/arches/app/views/search.py

```diff
@@ -192,6 +192,14 @@ def build_search_results_dsl(request):
     boolquery = Bool()
     boolfilter = Bool()
     if term_filter != '':
+        is_empty_temporal_filter = True

```

Comment:
@markdoub  this bit still gives problems with the Related Resources search: 
![screen shot 2017-10-06 at 16 31 32](https://user-images.githubusercontent.com/19243127/31285666-dceb2f20-aab3-11e7-9710-a42f4ee47957.png)

Reply:
Sorry, this doesn't look right. Line 194 should not be there...

Reply:
@markdoub  
```
        if len(temporal_filter['year_min_max']) != 0:
            is_empty_temporal_filter = False
        if len(temporal_filter['filters']) != 0:
            is_empty_temporal_filter = False

```

works on Rel Resources search, but haven't tested it platform-wide